nexus/src/authz/api_resources.rs

```diff
@@ -435,6 +435,10 @@ impl ProjectChild {
     pub fn id(&self) -> Uuid {
         self.resource_id
     }
+
+    pub fn project(&self) -> &Project {
+        &self.parent
+    }
```

Comment:
is this used?

Reply:
Yes, here:
https://github.com/oxidecomputer/omicron/blob/aa312ca5c18716c346828bf20a38b1c8680fda47/nexus/src/nexus.rs#L1557

Reply:
thanks. it didn't come up in my find because `nexus.rs` was collapsed